modules/client-configuration/pages/client-upgrades-uyuni.adoc

```diff
@@ -24,19 +24,19 @@ opensuse_leap15_4-uyuni-client
 // These are custom channels.
 // For more information, see xref:client-configuration:clients-opensuse.adoc[].
 
-. In the {productname} Server {webui}, navigate to menu:Software[Manage > Channels] and click the [systemitem]``Uyuni Client Tools for openSUSE Leap 15.4 (x86_64)`` channel name.
+. In the {productname} Server {webui}, navigate to menu:Software[Manage > Channels] and click the [systemitem]``Uyuni Client Tools for openSUSE Leap 15.5 (x86_64)`` channel name.
```

Comment:
Same here and for the other changes like this one.

Reply:
Changed